python/ray/data/dataset.py

```diff
@@ -2830,21 +2832,43 @@ def to_tf(
     ) -> "tf.data.Dataset":
         """Return a TF Dataset over this dataset.
 
-        The TF Dataset will be created from the generator returned by the
-        ``iter_batches`` method. ``prefetch_blocks`` and ``batch_size``
-        arguments will be passed to that method.
-
-        This is only supported for datasets convertible to Arrow records.
-
-        It is recommended to call ``.split()`` on this dataset if
-        there are to be multiple TensorFlow workers consuming the data.
```

Comment:
I didn't really understand this. What does this mean?

Reply:
This dates back to before Ray AIR or Ray Train had native Datasets support, when this was used with Horovod-on-Ray, so I think that this comment can be removed. Ray AIR/Train does the splitting for the user.